demos/common/samples/common.hpp

```diff
@@ -48,6 +48,11 @@ class ConsoleErrorListener : public InferenceEngine::IErrorListener {
     }
 };
 
+template <typename T, std::size_t N>
+inline std::size_t arraySize(const T (&)[N]) {
```

Comment:
I suggest adding `constexpr` and `noexcept` like in https://en.cppreference.com/w/cpp/iterator/size. And than `inline` is no longer needed.

Reply:
Alright, done. I initially thought the CentOS 7 version of GCC might be too old to support this, but I tested, and it isn't.